x/wasm-storage/keeper/msg_server.go

```diff
@@ -63,9 +69,9 @@ func (m msgServer) StoreOracleProgram(goCtx context.Context, msg *types.MsgStore
 			sdk.NewAttribute(types.AttributeOracleProgramHash, hashHex),
 		),
 	)
-	if err != nil {
-		return nil, err
-	}
+
+	// Increment the oracle program count.
+	OracleProgramCountMetric.Inc()
```

Comment:
Maybe it's running once in CheckTx and again in DeliverTx? If it's annoying to debug I would just drop it, this is a metric we can also easily get from the exploder.

Reply:
There was an easy fix :). The context has a method to check if it's in `CheckTx` or not.